week-4-practise/level-1/exercise-1.js

```diff
@@ -38,7 +38,46 @@ var writers = [
     alive: true
   }
 ];
-
+console.log(writers);
+console.log(writers.alive);

```

Comment:
This logs undefined, did you mean to do this?

Reply:
oh yea, i will adjust that